build.js

```diff
@@ -1,14 +1,11 @@
 'use strict';
 
-const { spawnSync } = require('child_process');
 const fs = require('fs-extra');
 const path = require('path');
 const glob = require('glob');
-const { DEFAULT_EXTENSIONS } = require('@babel/core');
 
 const SRC_DIR = 'src';
 const BUILD_DIR = 'build';
-const ELECTRON_BUILDER_ENV = 'electron-builder.env';
 
 if (process.argv[2] && process.argv[2] === 'clean') {
```

Comment:
yeah I think you're right. The pattern should probably be `if (process.argv && ....` but I think there will always be an `argv` property so that's excessive too.